lib/msal-browser/test/interaction_client/RedirectClient.spec.ts

```diff
@@ -3544,12 +3571,14 @@ describe("RedirectClient", () => {
                         correlationId: RANDOM_TEST_GUID,
                         postLogoutRedirectUri: TEST_URIS.TEST_REDIR_URI,
                     };
-                    expect(logoutUriSpy.calledWith(validatedLogoutRequest));
+                    expect(logoutUriSpy).toHaveBeenCalledWith(
+                        validatedLogoutRequest
+                    );
```

Comment:
This `expect` never resolves, and causes the test to fail.

Reply:
Never resolves or is never called? If it's never called it likely means the expect above failed or the logout failed. You can wrap this all in a try/catch and then pass the error into `done(error)` to receive a better error than a timeout

Reply:
It's called, but never resolves, and eventually the 5000 ms timeout error occurs

Reply:
It's likely failing rather than not resolving. Try the try/catch and done(error) approach above to get the real error